config/webpacker.yml

```diff
@@ -25,10 +25,10 @@ development:
   <<: *default
 
 # uncomment these to enable webpack:server
-#  dev_server:
-#    host: 0.0.0.0
-#    port: 8080
-#    https: false
+  dev_server:
```

Comment:
I still think it's a wrong idea to enable this here, especially since people will try it and find it doesn't work.